ETWAnalyzer/EventDump/DumpExceptions.cs

```diff
@@ -183,6 +182,12 @@ bool IsMatchingException(ExceptionEventForQuery arg)
 
             foreach (var ex in file.Extract.Exceptions.Exceptions.Where(IsMatchingException))
             {
+                ModuleDefinition exceptionModule = ShowModuleInfo ? file.Extract.Modules.Modules.Where(x => x.Processes.Contains(ex.Process)).Where(x => x.ModuleName.EndsWith(".exe", StringComparison.OrdinalIgnoreCase)).FirstOrDefault() : null;
```

Comment:
That was the old slow approach. I have added file.Extract.Modules.FindModule to make it faster. 

 ModuleDefinition exceptionModule = ShowModuleInfo ? file.Extract.Modules.FindModule(here you need to add the executable name without path, ex.Process) : null;